src/quo/components/text_combinations/page_top/view.cljs

```diff
@@ -112,10 +115,10 @@
         emojis))
 
 (defn- view-internal
-  [{:keys  [theme description input blur? input-props]
+  [{:keys  [theme description input blur? input-props container-style]
     emojis :emoji-dash
     :as    props}]
-  [rn/view
+  [rn/view {:style container-style}
```

Comment:
Add support for extra styles to page-top, some screens add an extra bottom margin of 8